Homework/css/styles.css

```diff
@@ -0,0 +1,95 @@
+
+body {	
+
+	background: rgb(197,222,234); /* Old browsers */
```

Comment:
@JsonLitz No shorter way to do it if you're concerned with cross-browser compatibility. I just wanted to make sure you had an understanding of what was going on instead of just copy / pasting from the net. One thing I can add is that as browsers improve, some properties that _used_ to require browser prefixing may no longer need it. But this looks fine to me. Good job here.